src/Breadcrumb.js

```diff
@@ -0,0 +1,47 @@
+import React, { cloneElement } from 'react';
+import classNames from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+import ValidComponentChildren from './utils/ValidComponentChildren';
+
+const Breadcrumb = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    /**
+     * CSS classes for the wrapper `ol` element
+     */
+    className: React.PropTypes.string,
+    /**
+     * HTML id for the wrapper `ol` element
+     */
+    id: React.PropTypes.string
```

Comment:
In that case it's to clarify that `id` and `className` are separate from what's attached to the wrapped `ul` element for documentation. You don't have that here, so it's not relevant.